decidim-core/app/controllers/concerns/decidim/use_organization_time_zone.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module Decidim
+  # Common logic to switch between locales.
+  module UseOrganizationTimeZone
+    extend ActiveSupport::Concern
+
+    included do
+      around_action :use_organization_time_zone
+      helper_method :organization_time_zone
+
+      # Sets the time zone used to user in the controller
+      # Returns nothing.
+      def use_organization_time_zone(&action)
+        Time.use_zone(organization_time_zone, &action)
```

Comment:
please, explain in the comment that this the `action` argument is a block that is wrapped with the organization's timezone.